WindowsServerDocs/get-started-19/servicing-channels-19.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Servicing Channels
-description: Explanation of Windows Server service channels - LTSC and SAC
+title: Windows Server servicing channels
+description: Explanation of Windows Server service channels: LTSC and SAC
```

Comment:
This actually needs to stay as the hyphen. Using the colon in the metadata confuses the build and formatting since it assumes you're adding a new line of metadata. Please change this back so I can accept the rest of your changes.

Reply:
Done.

Although, it is funny that your parser is not even as good GitHub's syntax highlighter. 😆 Oh, well, that's life I guess.